digital_benchmark/twitter_benchmark/data_parser.py

```diff
@@ -4,8 +4,9 @@
 class TwitterDataParser:
 
     @staticmethod
-    def parser_user_data(data):
+    def parser_user_data(data, user):
```

Comment:
@aleemnadeem rename the function to properly how what it is doing.
Refer to function naming guidelines to get better idea.